Vagrantfile

```diff
@@ -22,7 +23,25 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # Create a forwarded port mapping which allows access to a specific port
   # within the machine from a port on the host machine. In the example below,
   # accessing "localhost:8080" will access port 80 on the guest machine.
-  # config.vm.network "forwarded_port", guest: 80, host: 8080
+  config.vm.network "forwarded_port", guest: 3000, host: 3000
+  config.omnibus.chef_version = :latest
+  config.vm.provision "chef_zero" do |chef|
```

Comment:
[rubocop] Prefer single-quoted strings when you don't need string interpolation or special symbols.